src/riak_kv_get_core.erl

```diff
@@ -310,7 +310,7 @@ maybe_log_old_vclock(Results) ->
                         [] ->
                             ok;
                         _ ->
-                            lager:warning("Bucket: ~p Key: ~p should be rewritten to guarantee
+                            object:warning("Bucket: ~p Key: ~p should be rewritten to guarantee
```

Comment:
Yes, lager3 will do a parse transform on object: to route to the correct sink. Purpose of the change in rebar.config: {lager_extra_sinks, [object]}